<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/images/backgroundimage.png);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="NewAction"/>
    <addaction name="OpenAction"/>
    <addaction name="SaveAction"/>
    <addaction name="SaveAsAction"/>
    <addaction name="separator"/>
    <addaction name="UiAction"/>
    <addaction name="PrintfAction"/>
    <addaction name="separator"/>
    <addaction name="ExitAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="UndoAction"/>
    <addaction name="RedoAction"/>
    <addaction name="separator"/>
    <addaction name="CutAction"/>
    <addaction name="CopyAction"/>
    <addaction name="PasteAction"/>
    <addaction name="DeleteAction"/>
    <addaction name="separator"/>
    <addaction name="FindAction"/>
    <addaction name="InsteadAction"/>
    <addaction name="separator"/>
    <addaction name="SelectAllAction"/>
    <addaction name="DatetimeAction"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="action_W"/>
    <addaction name="fontAction"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="action_S_2"/>
    <addaction name="ColorAction"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="HelpAction"/>
    <addaction name="AboutAction"/>
    <addaction name="separator"/>
    <addaction name="ServiceAction"/>
    <addaction name="AboutAppAction"/>
    <addaction name="AboutQtAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="NewAction"/>
   <addaction name="OpenAction"/>
   <addaction name="SaveAction"/>
   <addaction name="separator"/>
   <addaction name="CopyAction"/>
   <addaction name="CutAction"/>
   <addaction name="PasteAction"/>
  </widget>
  <action name="NewAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/new.png</normaloff>:/new/prefix1/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N) </string>
   </property>
  </action>
  <action name="OpenAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/open.png</normaloff>:/new/prefix1/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)...</string>
   </property>
  </action>
  <action name="SaveAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/save.png</normaloff>:/new/prefix1/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="SaveAsAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/asave.png</normaloff>:/new/prefix1/images/asave.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)...</string>
   </property>
  </action>
  <action name="UiAction">
   <property name="text">
    <string>页面设计(&amp;U)...</string>
   </property>
  </action>
  <action name="PrintfAction">
   <property name="text">
    <string>打印(&amp;P)...</string>
   </property>
  </action>
  <action name="ExitAction">
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
  </action>
  <action name="UndoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/undo.png</normaloff>:/new/prefix1/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="CutAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/cut.png</normaloff>:/new/prefix1/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="CopyAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/copy.png</normaloff>:/new/prefix1/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="PasteAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/paste.png</normaloff>:/new/prefix1/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="DeleteAction">
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
  </action>
  <action name="FindAction">
   <property name="text">
    <string>查找(&amp;F)...</string>
   </property>
  </action>
  <action name="InsteadAction">
   <property name="text">
    <string>替换(&amp;I)...</string>
   </property>
  </action>
  <action name="GotoAction">
   <property name="text">
    <string>转到(&amp;G)...</string>
   </property>
  </action>
  <action name="SelectAllAction">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="action_F5">
   <property name="text">
    <string>&amp;F5</string>
   </property>
  </action>
  <action name="DatetimeAction">
   <property name="text">
    <string>时间/日期(D)</string>
   </property>
  </action>
  <action name="action_W">
   <property name="text">
    <string>自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="text">
    <string>字体(&amp;F)...</string>
   </property>
  </action>
  <action name="action_S_2">
   <property name="text">
    <string>状态(&amp;S)</string>
   </property>
  </action>
  <action name="HelpAction">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="AboutAction">
   <property name="text">
    <string>关于记事本(&amp;A)</string>
   </property>
  </action>
  <action name="RedoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/images/redo.png</normaloff>:/new/prefix1/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>重做(&amp;R)</string>
   </property>
  </action>
  <action name="ColorAction">
   <property name="text">
    <string>颜色(&amp;C)</string>
   </property>
  </action>
  <action name="ServiceAction">
   <property name="text">
    <string>百度一下(&amp;S)</string>
   </property>
  </action>
  <action name="AboutAppAction">
   <property name="text">
    <string>关于软件(&amp;M)</string>
   </property>
  </action>
  <action name="AboutQtAction">
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
